src/components/reusables/Filter/Filter.module.css

```diff
@@ -0,0 +1,42 @@
+@import "@digdir/design-system-tokens/dist/digdir/tokens.css";
+
+.popoverContent{
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    padding-left: 15px;
+    padding-right: 15px;
+}
+
+.modalHeader {
+    display: flex;
+    justify-content: space-between;
+}
+
+.optionSection{
```

Comment:
Vet ikke hvor jeg skal kommentere dette, men på macen min så lagger animasjonen som åpner popoveren for akkurat checkboxene og søkefeltet. Kan ikke animasjonen virke mot sin hensikt (å minke den kognitive belastningen) dersom det oppleves laggete?

Reply:
Fixed! 😊